app/assets/stylesheets/components/_card-auth.scss

```
@@ -0,0 +1,21 @@
+.card-auth {
+  @extend .card;
+
+  width: 45%;
+  margin: auto;
+  margin-top: 30px;
+  border-color: $secondary;
+
+  &__header {
+    @extend .card-header;
+
+    color: $white;
+    text-align: center;
+    background-color: $dark;
+  }
+  &__footer {
+    width: 40%;
+    padding-top: 20px;
```

Let's turn all `px` to `rem` 🙌 
```suggestion
    padding-top: rem(20px);
```

---

Resolved in a89ff14